playground/index.html

```diff
@@ -17,7 +17,20 @@
 <script src="../render.js"></script>
 <script>
     var canvas = document.getElementById('scratch-stage');
-    var renderer = new RenderWebGLLocal(canvas);
+    var fudge = 90;
+    var renderer = new RenderWebGL(canvas);
+
+    var drawableID = renderer.createDrawable();
+    renderer.updateDrawableProperties(drawableID, {
+        position: [0, 0],
+        scale: [100, 100],
+        direction: 90
+    });
+    var drawableID2 = renderer.createDrawable();
+    renderer.updateDrawableProperties(drawableID2, {
+        skin: 'https://cdn.assets.scratch.mit.edu/internalapi/asset/' +
+            '09dc888b0b7df19f70d81588ae73420e.svg/get/'
+    });
```

Comment:
Not a complaint about this PR, but this (or #27 in general) really makes me feel like we need to work on an asset store module.